features/step_definitions/comment_steps.rb

```diff
@@ -302,3 +302,24 @@
     click_link("Yes, delete!") # TODO: Fix along with comment deletion.
   end
 end
+
+When "I display comments" do
+  click_link("Comments")
+end
+
+When "I open the reply box" do
+  click_link("Reply")
+end
+
+When "I cancel the reply box" do
+  click_link("Cancel")
+end
+
+When /^I reply to the work comment "([^"]*)" on a new page$/ do |comment_content|
+  comment = Comment.find_by(comment_content: comment_content)
+  visit work_path(
+    comment.commentable,
+    add_comment_reply_id: comment.id,
+    show_comments: true
+  )
+end
```

Comment:
Was originally:
```rb
new_window = window_opened_by do
  find(:link, "Reply").click(:control)
end
switch_to_window(new_window)
```
To be closer to real life behavior. But I encountered issues with domains (see below) and moved to a more direct approach.

Reply:
I ran into the same issue with [domains in testing](https://github.com/otwcode/otwarchive/pull/4558/files#diff-bff3e97645581265ef40912813188da582223b750b875fa0c24c1d4f40f1e948R2) and it's due to the use of URLs rather than paths. You might be able to resolve it by modifying this [helper](https://github.com/otwcode/otwarchive/blob/801ce0931c0dbb144f9631eed5e9cf743a6c8d00/app/helpers/comments_helper.rb#L192)